js/ccf-app/src/global.ts

```diff
@@ -242,6 +242,12 @@ export interface CCFRpc {
    * The default is `false`.
    */
   setApplyWrites(force: boolean): void;
+
+  /**
+   * Set a claims digest to be associated with the transaction if it succeeds. This
+   * digest can later be accessed from the receipt, and expanded into a full claim.
+   */
+  setClaimsDigest(digest: ArrayBuffer): void;
```

Comment:
The docstring should say what kind of hash this is, sha-256? Or at least how to figure it out if it's not mentioned here directly.

Reply:
It's sha256 right now, I will update the docstring and see if I can add a ref to ccf.digest()